install_config/configuring_ephemeral.adoc

```diff
@@ -56,45 +58,44 @@ apiServerArguments:
    - LocalStorageCapacityIsolation=true
 ...
 ----
-```
-
-. Edit the ConfigMap for all nodes to enable the LocalStorageCapacityIsolation on the command line.  You can identify the ConfigMaps that need to be edited as follows:
 
-```
+. Edit the ConfigMap for all nodes to enable the LocalStorageCapacityIsolation on the command line. You can identify the ConfigMaps that need to be edited as follows:
```

Comment:
I think this needs to use more steps, maybe like:
. Obtain a list of the nodes to add the `LocalStorageCapacityIsolation` parameter to:
+
command & output
. For each node listed, add the `LocalStorageCapacityIsolation` parameter:
.. Edit the ConfigMap:
+
$ oc edit cm node-config-master -n \<openshift-node> \<1>
\<1> \`<openshift_node> is the name of the node to modify.
.. Add the feature gate to the node:
\* If there is already a `feature-gates:` declaration...
\* If there is no `feature-gates:` declaration...